containers/clang.docker/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM debian:buster
+MAINTAINER ArangoDB Hackers <hacker@arangodb.com>
+
+#set proper shell
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# install base dependencies
+RUN apt-get update && apt-get install -y -q --no-install-recommends \
+        apt-transport-https \
+        build-essential \
+        ca-certificates \
+        gnupg2 \
+        libssl-dev \
+        wget \
+        curl \
+        tar \
+        git \
+        rsync \
+        parallel \
+        tree \
+        jq \
+        && apt-get clean
+
+## install packages for python
```

Comment:
@ObiWahn can you put this in one apt-get?

Do we need both python and python3?

Reply:
Python2 is about to expire ( https://pythonclock.org/ ). Unfortunately our build still requires this versions. I think we should remove the python2 package as soon as possible.

Reply:
fix it ;-P https://github.com/refack/GYP3

Reply:
It will not be fixed here. But if we have to change we will not need to build a new container...

Reply:
@dothebart please do not bring up the 2% increase in container size :)